src/liballoc/arc.rs

```diff
@@ -71,6 +71,12 @@ const MAX_REFCOUNT: usize = (isize::MAX) as usize;
 /// does not use atomics, making it both thread-unsafe as well as significantly
 /// faster when updating the reference count.
 ///
+/// Note: as for `Rc<T>`, the inherent methods defined on `Arc<T>` are all
```

Comment:
Can you drop the "as for `Rc<T>`" here? I think it's more confusing than it's worth.